clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -41,7 +44,8 @@
     HumanReadable {
 
   public static final boolean DEFAULT_AUTOCREATE = false;
-  private final URI clusterUri;
+  private final ConnectionSource connectionSource;
+  private final String cacheManagerName;
```

Comment:
I think this should be renamed to `entityName` or `clusterTierManagerEntity` or something like that.
Because this is confusing since there is no cache manager name, and in management, we have a cache manager alias attribute on the Management configuration ;-)